.github/workflows/config/testing/x64/evm.mas.json

```diff
@@ -0,0 +1,25 @@
+{
+  "root": "/Users/runner/work/electron/electron/",
+  "remotes": {
+    "electron": {
```

Comment:
I think this is fine for now but maybe we can clean up these file duplicates as a next step 🤔